themes/default/content/docs/pulumi-cloud/deployments/reference.md

```diff
@@ -197,6 +197,20 @@ Once the GitHub app has been installed, the deployment settings for a stack can
 - The GitHub app may only be installed by a GitHub *and* Pulumi admin.
 - Currently, there is a 1 to 1 mapping between GitHub organizations and Pulumi organizations.
 
+#### Private Dependency Packages
+
+If you want to private GitHub repositories, including private packages and resources, the Pulumi program needs access to the SSH private key granted permissions to connect with Github. Failure to do so may result in remote deployments being unable to access those private artifacts, and the deployment may fail.
+
+As a current fix, using the following will allow you to create a private key and allow access to GitHub using SSH to pull down the appropriate artifacts properly:
+  
+1. Add the following code into the `Pre-run commands` and toggle on `Skip automatic dependency installation step` in Advanced Settings:
```

Comment:
It would be helpful to have this code in text so folks can just copy-paste rather than having to retype instructions from an image.

Reply:
Added, thanks